assets/base.css

```diff
@@ -2830,16 +2837,28 @@ details-disclosure > details {
 
 .grid--collapsed.contains-content-container .content-container,
 .grid--collapsed.contains-media .global-media-settings {
+  border-radius: var(--top-left-radius) var(--top-right-radius) var(--bottom-right-radius) var(--bottom-left-radius);
   margin-bottom: var(--collapsed-offset-y);
   margin-right: var(--collapsed-offset-x);
```

Comment:
here I noticed that for multicolumn, when it's not set to swipe on mobile, the blocks gets too wide due to a negative margin: 
![](https://screenshot.click/22-01-alrsu-qsxvg.png)

Reply:
This was only on multicolumn right? The issue looked to be a width: 100% on multicolumn list. I was hesitant to remove it in this PR, but it really didn't seem like it was needed for any current multicolumn configurations.